test/Integration/GetWorks.ts

```diff
@@ -0,0 +1,96 @@
+/* tslint:disable:no-relative-imports */
+import { Claim, isClaim, Work } from "@po.et/poet-js";
+import {
+  AsyncTest,
+  Expect,
+  SetupFixture,
+  TestCase,
+  TestFixture
+} from "alsatian";
+
+import { AStudyInScarlet, TheMurdersInTheRueMorgue, TheRaven } from "../Claims";
+import { Client } from "./Helper";
+
+@TestFixture("GET /works")
+export class GetWorks {
+  private client: Client;
+
+  @SetupFixture
+  public setupFixture() {
+    this.client = new Client();
+  }
+
+  @AsyncTest()
+  @TestCase()
+  async getWorksShouldSucceed() {
+    const response = await this.client.getWorks();
+
+    Expect(response.status).toBe(200);
+    Expect(response.ok).toBeTruthy();
+  }
+
+  @AsyncTest()
+  @TestCase()
+  async getWorksShouldReturnAnArray() {
+    const response = await this.client.getWorks();
+
+    Expect(response.status).toBe(200);
+    Expect(response.ok).toBeTruthy();
+
+    const claims = await response.json();
+
+    Expect(claims).toBeDefined();
+    Expect(Array.isArray(claims)).toBeTruthy();
+  }
+
+  @AsyncTest()
+  @TestCase()
+  async getWorksShouldReturnClaims() {
+    const response = await this.client.getWorks();
+
+    Expect(response.status).toBe(200);
+    Expect(response.ok).toBeTruthy();
+
+    const claims = await response.json();
+    const allElementsAreClaims = !claims.find(
+      (claim: Claim) => !isClaim(claim)
+    );
+
+    Expect(allElementsAreClaims).toBeTruthy();
+  }
+
+  @AsyncTest()
+  @TestCase()
+  async getWorksShouldReturnExpectedFields(
```

Comment:
This test fails after I let the node run for long enough and I'm assuming it is because there are some claims that aren't valid being posted?